crawlerr/services/rag_service.py

```
@@ -73,6 +162,215 @@ def chunk_text(self, text: str, metadata: Dict[str, Any] = None) -> List[Dict[st
+        
+    def _find_word_boundary(self, search_text: str, ideal_end: int) -> Optional[int]:
+        """Find word boundary."""
+        return find_word_boundary(search_text, ideal_end)
+        
+    def _find_semantic_boundary(
+        self, 
+        full_text: str, 
+        chunk_text: str, 
+        start_token_idx: int, 
+        end_token_idx: int, 
+        tokens: List[int],
```

Ruff (0.12.2)

Unused method argument: `tokens`

(ARG002)